packages/bsky/src/data-plane/bsync/index.ts

```diff
@@ -210,6 +221,61 @@ const handleNotificationPreferencesOperation = async (
   return handleGenericOperation(db, req, now)
 }
 
+const handleSubjectActivitySubscriptionOperation = async (
+  db: Database,
+  req: PutOperationRequest,
+  now: string,
+) => {
+  const { actorDid, key, method, payload } = req
+
+  const parsed: SubjectActivitySubscription = JSON.parse(
+    Buffer.from(payload).toString('utf8'),
+  )
+  const { activitySubscription } = parsed
+  const { post, reply } = activitySubscription
+  // We only keep a record if enabled.
+  const enabled = isActivitySubscriptionEnabled(activitySubscription)
+
+  if (method === Method.CREATE) {
+    if (enabled) {
+      return db.db
+        .insertInto('activity_subscription')
+        .values({
+          creator: actorDid,
+          subjectDid: parsed.subject,
+          key,
+          createdAt: now,
+          indexedAt: now,
+          post,
+          reply,
+        })
+        .execute()
+    }
+    return
+  }
+
+  if (method === Method.UPDATE && enabled) {
+    return db.db
+      .updateTable('activity_subscription')
+      .where('creator', '=', actorDid)
+      .where('subjectDid', '=', parsed.subject)
+      .where('key', '=', key)
```

Comment:
`(creator, subjectDid)` and `key` are each unique.  Is there a reason both are needed?  What happens if an update for a given `key` changes the `creator` or `subjectDid`?